bottlelib/Data/List/Utils.hs

```diff
@@ -34,7 +34,7 @@ insertAt n x xs = take n xs ++ x : drop n xs
 isLengthAtLeast :: Int -> [a] -> Bool
 isLengthAtLeast n list
     | n <= 0 = True
-    | otherwise = list ^? Lens.ix (n-1) & Lens.has Lens._Just
+    | otherwise = Lens.element (n-1) `Lens.notNullOf` list
```

Comment:
Yeah that code is indeed a bit strange, had a look and came up with a more concise version now -
https://github.com/lamdu/lamdu/commit/fd3468440ceb9e3024598e8f03e19be44bfdccdb

Btw do you prefer `notNullOf` over `has`?